api/test/wfl/integration/executor_test.clj

```diff
@@ -67,15 +67,16 @@
 ;; Mocks
 
 ;; Snapshot and snapshot reference mocks
-(def ^:private snapshot
-  {:name   "test-snapshot-name"
+(def ^:private snapshot-name "test-snapshot-name")
+(def ^:private snapshot-table-name "snapshot-table")
+(defn ^:private mock-datarepo-snapshot [& _]
+  {:name   snapshot-name
    :id     (str (UUID/randomUUID))
-   :tables [{:name "table"}]})
+   :tables [{:name snapshot-table-name}]})
```

Comment:
This is fine, but as a reminder to discuss this …
Consider working with an expected data structure instead of breaking this down to strings.